### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Pages - 
- Landing Page

Components - 
- Navbar + Sidebar
- Footer
- Routes for navigation

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]

**Review by Prabhav-Sharma — APPROVED** [review]
Love the design and the attention to detail in transitions

**Review by JowelTisso — APPROVED** [review]
* Clean code
* Good work with scss
* Good work with extracting all routes
* Looks Good

File: src/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import SignIn from "./Auth/Signin";
+import SignUp from "./Auth/Signup";
+
+import Home from "./Home/Home";
+import VideoListing from "./VideoListing/VideoListing";
+
+export { Home, VideoListing, SignIn, SignUp };
```
Comment (deepak29-git) [inline src/pages/index.js:7]: you are importing all files in index.js and export it from here , this is good approach ,I liked it

**Review by deepak29-git — APPROVED** [review]

**Review by sharathnair9999 — COMMENTED** [review]
Wow. Code looks so clean and the UI is awesome

**Pull request merged**